AutomatticTracks/build.gradle

```diff
@@ -20,23 +20,22 @@ dependencies {
 
     testImplementation 'junit:junit:4.13.2'
     testImplementation 'org.assertj:assertj-core:3.19.0'
-    testImplementation 'org.mockito.kotlin:mockito-kotlin:2.2.11'
-    testImplementation 'org.mockito:mockito-inline:3.8.0'
+    testImplementation 'org.mockito.kotlin:mockito-kotlin:4.0.0'
+    testImplementation 'org.mockito:mockito-inline:4.6.1'
     testImplementation "org.jetbrains.kotlinx:kotlinx-coroutines-test:1.6.4"
 
-    androidTestImplementation 'androidx.test.ext:junit:1.1.2'
-    androidTestImplementation 'androidx.test:runner:1.3.0'
+    androidTestImplementation 'androidx.test.ext:junit:1.1.3'
+    androidTestImplementation 'androidx.test:runner:1.4.0'
 }
 
 android {
-    compileSdkVersion 30
+    compileSdkVersion 33
 
     defaultConfig {
-        versionName "2.1.0"
         minSdkVersion 21
-        //noinspection OldTargetApi
-        targetSdkVersion 30
+        targetSdkVersion 33
         buildConfigField "String", "SENTRY_TEST_PROJECT_DSN", project.properties.get("sentryTestProjectDSN")
+        buildConfigField "String", "VERSION_NAME", "\"2.1.0\""
```

Comment:
It was a constant before, I'm not changing anything here. But it's invalid. More details internally: pdnsEh-IK-p2

Reply:
Praise (❤️): Yeap, yeap, thank you for this change and for providing more details on that. Library modules no longer have the `versionName/Cope` configuration, only application modules have that with the latest AGP.